package.json

```diff
@@ -9,42 +10,50 @@
   ],
   "author": "Dominic Elm",
   "main": "dist/index.js",
+  "scripts": {
+    "build": "rm -rf dist && tsc -b",
+    "test": "vitest run",
+    "test:watch": "vitest",
+    "lint": "eslint \"{src,test}/**/*.{js,ts,json,jsonc}\"",
+    "preversion": "npm test",
+    "postversion": "git push && git push --tags",
+    "prepack": "npm run build"
+  },
+  "engines": {
+    "node": "^18.0.0 || ^20.0.0"
+  },
   "files": [
     "dist",
     "docs",
     "package.json",
     "README.md",
     "LICENSE"
   ],
-  "scripts": {
-    "build": "tsc -b",
-    "test": "jest",
-    "preversion": "npm test",
-    "postversion": "git push && git push --tags",
-    "prepack": "npm run build"
-  },
   "dependencies": {
-    "@typescript-eslint/eslint-plugin": "5.59.7",
-    "@typescript-eslint/parser": "^5.59.7",
-    "@typescript-eslint/utils": "^5.59.7",
-    "common-tags": "^1.8.0",
-    "eslint": "^8.11.0",
-    "eslint-config-prettier": "^8.5.0",
-    "eslint-plugin-jsonc": "^2.6.0",
-    "eslint-plugin-prettier": "^4.0.0"
+    "@stylistic/eslint-plugin-ts": "^2.2.2",
+    "@typescript-eslint/eslint-plugin": "^8.0.0-alpha.30",
+    "@typescript-eslint/parser": "^8.0.0-alpha.30",
+    "@typescript-eslint/utils": "^8.0.0-alpha.30",
```

Comment:
Why do we need the alpha versions?

Reply:
Yea unfortunately we do need them because only those support the new flat config format.

Reply:
But I talked to James Henry and they will release the real version very soon, since it's already a release candidate.